.gitignore

```diff
@@ -0,0 +1,9 @@
+*.abcl
+*.fasl
```

Comment:
Are these from your own experience? I'm looking at the list here to get an idea re: typical lisp gitignore https://github.com/github/gitignore/blob/main/CommonLisp.gitignore

Reply:
No I haven't come across these personally. It was just part of the boilerplate I got from using `cl-project` to init a new project.

https://lispcookbook.github.io/cl-cookbook/getting-started.html#creating-a-new-project